packaging/Dockerfile

```diff
@@ -15,13 +15,22 @@ ARG branch=release
 
 # Install required system packages to build TeleSculptor
 # Everything else is built from Fletch
-RUN apt-get update && apt-get install -y --no-install-recommends \
-build-essential file git cmake \
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
```

Comment:
Note: I had to add `DEBIAN_FRONTEND=noninteractive` here because sphinx requires `tzdata` which requires an interactive configuration of your timezone which hangs the docker build if you don't make it `noninteractive`.